api/libproject/devstart.rb

```diff
@@ -18,15 +19,20 @@ def initialize(command)
         self.project = project
       end
       parser.on("--account [ACCOUNT]",
-           "Account to use when creating credentials (your.name@pmi-ops.org)") do |account|
+           "Account to use when creating credentials (your.name@pmi-ops.org); "\
+           "use this or --creds_file") do |account|
         self.account = account
       end
+      parser.on("--creds_file [CREDS_FILE]",
+           "Path to a file containing credentials; use this or --account.") do |creds_file|
+        self.creds_file = creds_file
+      end
     end
   end
 
   def parse args
     self.parser.parse args
-    if self.project == nil || self.account == nil
+    if self.project == nil || !((self.account == nil) ^ (self.creds_file == nil))
```

Comment:
I ran it like this:
```
./project.rb connect-to-cloud-db --project all-of-us-workbench-test --account david.mohs@pmi-ops.org
```
and it failed like this:
```
+ gsutil cp gs://all-of-us-workbench-test-credentials/vars.env /var/folders/hf/4v37bq753p3f1fvz953lgm2h1rlccx/T/all-of-us-workbench-test-vars.env20170823-7135-i246pe
AccessDeniedException: 403 dmohs@broadinstitute.org does not have storage.objects.list access to bucket all-of-us-workbench-test-credentials.
```
However, the goal here is for Circle to run the migrations, right? If so, I think we should focus on that use case. Circle is a different-enough environment that I think we can special-case it, and I don't think it needs to be exposed for developers to run (at least while we focus on Circle).

Here is what we did in the UI:
https://github.com/all-of-us/workbench/blob/master/.circleci/config.yml#L52

That technique allows us to share common code, but the stuff that Circle needs doesn't clutter up the help output for commands developers are likely to run. I suggest we start with that (make Circle happy) and then worry about exposing these commands for developers (possibly using docker instead of requiring mysql) later.